src/Cmts.ml

```diff
@@ -490,10 +490,35 @@ let fmt_cmt (conf : Conf.t) ~fmt_code (cmt : Cmt.t) =
             | _, None -> str line $ fmt "*)"
             | _, Some _ -> str line $ fmt "@,*") )
   in
-  let fmt_non_code (cmt : Cmt.t) =
+  let fmt_unwrapped_cmt ({txt= s; loc} : Cmt.t) =
+    let lines = String.split_lines s in
+    let first_line_empty =
+      List.length lines > 0 && String.is_empty (List.hd_exn lines)
+    in
+    let begins_line = Source.begins_line t.source loc ~ignore_spaces:false in
+    if String.contains s '\n' && (not begins_line) && not first_line_empty
```

Comment:
What is the reason for this condition ?

Reply:
- the first one is obviously because we only want to do the special treatment when there are `\n` inside (not using the box and so on)
- the second one: breaking comments that begin at the beginning of the line have no incidence of the associated code (the issue was that the code was not broken when the comment was broken, but if the comment is at column 0 it is very likely that the associated code is already either a one-liner or broken), so no different treatment to avoid a huge diff
- the last one because if the first line of a comment is empty we don't want to indent the following lines relatively to this line

Reply:
Thanks, that makes sense.